howto/HACKME.md

```diff
@@ -148,3 +152,51 @@ dist/Debug/GNU-Linux/amos-ss17-proj4 can0
 ```
 
 To modify the build system best open the project in Netbeans (see above).
+
+## Set-Up on Virtual Machine
+If the simulator should be installed on a virtual linux machine just do the following steps to get the simulator running.
+
+ISO-TP
```

Comment:
What is this line for?

Reply:
I misunderstood something there. I will change that and just describe what should be done to load the modules automatically.